WordPress/Classes/Utility/Analytics/WPAnalyticsEvent.swift

```diff
@@ -361,4 +384,28 @@ extension WPAnalytics {
         WPAnalytics.trackString(event.value, withProperties: mergedProperties)
     }
 
+    /// Track a Reader event in Obj-C
+    ///
+    /// This will call each registered tracker and fire the given event
+    /// - Parameter event: a `String` that represents the Reader event name
+    /// - Parameter properties: a `Hash` that represents the properties
+    ///
+    @objc static func trackReaderEvent(_ event: WPAnalyticsEvent, properties: [AnyHashable: Any], service: ReaderTopicService) {
+        var props = properties
+        props[WPAppAnalyticsKeySubscriptionCount] = service.allSiteTopics()?.count ?? 0
```

Comment:
Overall, singletons are always a bad idea. But I was thinking that perhaps this might be a good use case.

The idea would be:

* Store the number of followed sites in a `static` property to use it here
* Update this number every time the topics are updated or the user follow/unfollow a site

This will avoid performing a FetchRequest every time a Reader event is fired (and there are a lot of them being fired).

What do you think?